packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -33,19 +33,29 @@ export function write_tsconfig(config) {
 		include.push(config_relative(`${dir}/**/*.svelte`));
 	});
 
+	/** @type {Record<string, string[]>} */
+	const paths = {};
+	// $lib should be moved into a default value for `config.kit.alias`, for now we have this secondary
+	// object to handle the default $lib
```

Comment:
I'd probably drop this comment as a duplicate of the one in `core/utils.js`. I also made a wording suggestion on that one to move some of the wording here to that location since it seems more like the core location for this code
```suggestion
```